engine/runtime-parser/src/test/java/org/enso/compiler/core/IrPersistanceTest.java

```diff
@@ -470,4 +501,11 @@ public record ServiceSupply(Service supply) {}
 
   @Persistable(clazz = IdHolder.class, id = 432876)
   public record IdHolder(UUID id) {}
+
+  @Persistable(clazz = RefHolder.class, id = 436872)
+  public record RefHolder(Persistance.Reference<UUID> id) {
```

Comment:
Have a class with field `Persistance.Reference` and write/generate `Persistance` instance for it. In this case we generate the class with `@Persistable(clazz = RefHolder.class...` annotation.